account_payment_credit_card/__manifest__.py

```diff
@@ -1,14 +1,14 @@
-# Copyright 2017 Ursa Information Systems <http://www.ursainfosystems.com>
+# Copyright 2017 Open Source Integrators <http://www.opensourceintegrators.com>
 # License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl).
 
 {
     "name": "Credit Card Payments",
     "summary": "Add support for credit card payments",
     "version": "11.0.1.0.0",
     "license": "AGPL-3",
-    "author": "Ursa Information Systems, Odoo Community Association (OCA)",
+    "author": "Open Source Integrators, Odoo Community Association (OCA)",
     "category": "Accounting & Finance",
-    "maintainer": "Ursa Information Systems",
+    "maintainer": "Open Source Integrators",
```

Comment:
This key is incorrect. You should put here a GitHub login or remove it.

Reply:
@pedrobaeza The key for Github logins is `maintainers` ;)

https://github.com/OCA/mis-builder/blob/10.0/mis_builder/__manifest__.py#L40

Reply:
OK, then what is the sense of putting a redundant information in a non used key?

Reply:
@pedrobaeza `maintainer` comes from Odoo and contains a company name. `maintainers` was added by @sbidoul and contains Github logins.

Reply:
OK, I won't insist more, but I still think it's a useless key not shown anywhere.